packages/service-library/src/servicelib/long_running_tasks/_redis_store.py

```diff
@@ -23,17 +23,21 @@ def _load_from_redis_hash(data: dict[str, str]) -> dict[str, Any]:
     return {k: json_loads(v) for k, v in data.items()}
 
 
+def to_redis_namespace(namespace: LRTNamespace) -> str:
+    return namespace.upper()
+
+
 class RedisStore:
     def __init__(self, redis_settings: RedisSettings, namespace: LRTNamespace):
```

Comment:
MINOR: what about "highlighting" a bit more that `lrt_namespace != redis_namespace` but calling all these argumrnts `lrt_namespace`?